src/client/resources/web/css/main.css

```diff
@@ -8,6 +8,25 @@ body {
     overflow: hidden;
 }
 
+#hover-container {
+    font-family: 'JetBrains Mono', monospace;
+
+    display: none;
+    z-index: 1000;
+    position: absolute;
+    top: 0;
+    left: 0;
+
+    border: 2px solid #0000aa;
```

Comment:
Let's use the actual colors here. For the border that is `#25035C` for the outline and background `#130512`.

![image](https://github.com/user-attachments/assets/83733dc9-998e-40ad-ba5f-91821e7ecf86)
![image](https://github.com/user-attachments/assets/6479a8af-f539-4fc2-9d42-79e27cd68eeb)